src/components/form/templates/grid-layout.ts

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { FormLayoutOptions, GridLayoutOptions } from '../form.types';
+
+const MAX_COLUMNS = 5;
+const MAX_COLUMN_WIDTH = 15;
+const REM_TO_PX = 16;
+const COLUMN_COUNT = '--number-of-columns';
+
+export class GridLayout extends React.Component<LayoutProps, LayoutState> {
```

Comment:
Right now this layout just works in the form, but we could probably quite easily move this implementation to the `limel-grid` component instead and use that one in here. The reason for not doing it right now was that I did not realize that until later and it might also introduce breaking changes in the existing component. We can always do it later if we decide not to do it now :slightly_smiling_face:

Reply:
I don't really follow. You mean we should move these new layout possibilities/options (spaning and dense layout) to `limel-grid` instead?

Reply:
I'm just saying we could, there's not much difference between using them outside or inside of a form :slightly_smiling_face:

Reply:
That's right. Specially because that way of `grid-area` template thingy is quite hard to work with, isn't responsive, and not so sustainable. 

But how can can we transition them to use this new way? It's a breaking change and people won't be so happy if we change it I guess?

Maybe we can deprecate that and make a new limel-grid component with all these features? 

It'd also be nice to use it on the CRM dashboard

Reply:
(In other words, I think we should keep this separate from limel-grid for now 🙂)